backend/tests/Tests/SqlCompiler.Tests.fs

```diff
@@ -184,6 +184,57 @@ let inlineWorksWithNested =
     return! result |> Ply.toTask
   }
 
+let inlineWorksWithPackageFunctionsSqlBinOp =
+  testTask "inlineWorksWithPackageFunctionsSqlBinOp" {
+    let! state =
+      executionStateFor
+        (System.Guid.NewGuid())
+        false
+        false
+        Map.empty
+        Map.empty
+        Map.empty
+        Map.empty
+    let fns = ExecutionState.availableFunctions state
+    let! expr =
+      p
+        "let x = true in (let y = false in (PACKAGE.Darklang.Stdlib.Bool.and_v0 x y))"
+
+    let! expected = p "true && false"
+    let result =
+      uply {
+        let! result = C.inline' fns "value" Map.empty expr
+        return Expect.equalExprIgnoringIDs result expected
+      }
+    return! result |> Ply.toTask
+  }
+
+let inlineWorksWithPackageFunctionsSqlFunction =
+  testTask "inlineWorksWithPackageFunctionsSqlFunction" {
+    let! state =
+      executionStateFor
+        (System.Guid.NewGuid())
+        false
+        false
+        Map.empty
+        Map.empty
+        Map.empty
+        Map.empty
+    let fns = ExecutionState.availableFunctions state
+    let! expr =
+      p
+        """let x = "package" in (let y = "e" in (PACKAGE.Darklang.Stdlib.String.replaceAll_v0 x y "es"))"""
+
+    let! expected = p """Builtin.String.replaceAll_v0 "package" "e" "es" """
+    let result =
```

Comment:
> add tests in SqlCompiler.Tests that test the inlining. I think we should have a test where we can actually see the inlined code, if possible. (a pattern match against the inlined expr might be the best we can do)

I've added a couple of tests, but I am not sure about what you meant by "a pattern match against the inlined expr" could you please elaborate?

Reply:
This is better actually